### PR TITLE
enable the cpu manager on nodes not on the master

### PR DESCRIPTION
**Pull request by vladikr**
Allow starting k8s CPU manager on any node besides master.
This is needed for testing features that uses k8s CPU manager

Signed-off-by: Vladik Romanovsky <vromanso@redhat.com>

### CODE REVIEW

**Comment by vladikr** [comment]
Still working on a way to enable the cpu manager on the openshift

**Comment by vladikr** [comment]
This PR is required to enable functional testing for https://github.com/kubevirt/kubevirt/pull/1381

File: os-3.10/scripts/node01.sh [hunk 1]
```diff
@@ -74,3 +74,29 @@ done
 
 /usr/bin/oc create -f /tmp/local-volume.yaml
 
+# enable the CPU manager on compute nodes, besides master
+# updating it on the master as nodes cannot update comfigmaps
+
+sed -i 's/kubeletArguments:/kubeletArguments:\n  cpu-manager-policy:\n  - static\n  system-reserved:\n  - cpu=500m\n  kube-reserved:\n  - cpu=500m/' /etc/origin/node/node-config.yaml
+sed -i 's/feature-gates:/feature-gates:\n  - CPUManager=true/' /etc/origin/node/node-config.yaml
+
+/usr/bin/oc create cm node-config-compute -n openshift-node --from-file=/etc/origin/node/node-config.yaml -o yaml --dry-run |oc replace -f -
```
Comment (cynepco3hahue) [inline os-3.10/scripts/node01.sh]: nice

File: os-3.10/scripts/node01.sh [hunk 2]
```diff
@@ -74,3 +74,29 @@ done
...
+        - file:
+            state: absent
+            path: /var/lib/origin/openshift.local.volumes/cpu_manager_state
+        - service:
+            name: origin-node
+            state: restarted
+            enabled: yes
+EOF
+
+sleep 200
```
Comment (cynepco3hahue) [inline os-3.10/scripts/node01.sh]: Why do you need sleep here?

Comment (vladikr) [reply os-3.10/scripts/node01.sh]: I wish I didn't have to.
openshift runs a script[1] on its start and run it in the background.
It's basically generates an updated config from a configmap and restarts kubelet.
Overall, if it doesn't succeed it sleeps 180 - no matter what I did I couldn't enable the cpu manager
before sleep 200.

I will open a bug for origin and it's ansible so they will allow us to start the cpu manager in a more efficient way. 

[1] https://paste.fedoraproject.org/paste/sAPpMV0XJDlBkKMKqVvfrA

File: README.md [hunk 3]
```diff
@@ -22,12 +22,12 @@
 * `kubevirtci/centos:1804_02`: `sha256:70653d952edfb8002ab8efe9581d01960ccf21bb965a9b4de4775c8fbceaab39`
 * `kubevirtci/os-3.9.0:`: `sha256:234b3ae5c335c9fa32fa3bc01d5833f8f4d45420d82a8f8b12adc02687eb88b1`
 * `kubevirtci/os-3.9.0-crio:`: `sha256:107d03dad4da6957e28774b121a45e177f31d7b4ad43c6eab7b24d467e59e213`
-* `kubevirtci/os-3.10.0:`: `sha256:cdc9f998e19915b28b5c5be1ccc4c6fa2c8336435f38a37855f75b206977cbc2`
+* `kubevirtci/os-3.10.0:`: `sha256:faa467495207af8faa9214b1bf8adabf6161fab7f4da11b63efa41610a3ff0ab`
 * `kubevirtci/os-3.10.0-crio:`: `sha256:36cc53fd72fbd247810b38618c0f3fa058c17bcb6744b0512193eba0728ebf05`
```
Comment (cynepco3hahue) [inline README.md]: What about crio?

**Comment by vladikr** [comment]
Ah, I wasn't sure. I think I miss understood you yesterday :)

I'll add it, I'm just away at the moment.

File: README.md (hunk 3 above)
Comment (vladikr) [reply README.md]: Added :)

**Comment by fabiand** [comment]
I'm merging this, to keep the (this) code in sync with the already pushed images

**Pull request merged**